trace_decoder/Cargo.toml

```diff
@@ -28,7 +28,7 @@ thiserror = { workspace = true }
 
 # Local dependencies
 # TODO: update decoder to take local versions again
-mpt_trie = "0.1.0"
+mpt_trie = "0.2.0"
```

Comment:
This is unrelated I believe?

Reply:
Otherwise trace decoder was not even compiling